File: Objetivo 1/Exemplo1_PHP_MVC/model/Carro.php
<?php

/**
 * Description of Carro
 *
 * @author vpsil
 */
class Carro {
    //atributos
    public $ano;
    public $modelo;
    public $marca;
    
    //construtor de objetos
    public function __construct($ano, $modelo, $marca) {
        $this->ano = $ano;
        $this->modelo = $modelo;
        $this->marca = $marca;
    }
    
    public function __toString() {
        return "Carro[marca=$this->marca, modelo=$this->modelo, marca=$this->marca]";
    }
}

File: Objetivo 1/Exemplo1_Java_MVC/src/model/Carro.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package model;

/**
 *
 * @author vpsil
 */
public class Carro {
    //atributos
    public int ano;
    public String modelo;
    public String marca;
    
    //construtor de objetos
    public Carro(){}
    
    public Carro(int ano, String modelo, String marca) {
        this.ano = ano;
        this.modelo = modelo;
        this.marca = marca;
    }

    @Override
    public String toString() {
        return "Carro{" + "ano=" + ano + ", modelo=" + modelo + ", marca=" + marca + '}';
    }
}
